sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -727,6 +736,32 @@ Parameters:
 
 Returns: A new executable graph object which can be submitted to a queue.
 
+|
+[source,c++]
+----
+void
+print_dot_graph(const char* path, debug_print_flags  flags) const;
```

Comment:
Any reason to prefer a C-string here over a `std::string` for `path`?

Reply:
No. It is now a std::string.